src/svgcontext.ts

```diff
@@ -538,8 +543,8 @@ export class SVGContext implements RenderContext {
       const y1 = y + radius * Math.sin(startAngle + Math.PI);
       // There's no way to specify a completely circular arc in SVG so we have to
       // use two semi-circular arcs.
-      this.path += `M${x0} ${y0} A${radius} ${radius} 0 0 0 ${x1} ${y1} `;
-      this.path += `A${radius} ${radius} 0 0 0 ${x0} ${y0}`;
+      this.path += `M${round3(x0)} ${round3(y0)} A${radius} ${radius} 0 0 0 ${round3(x1)} ${round3(y1)} `;
+      this.path += `A${round3(radius)} ${round3(radius)} 0 0 0 ${round3(x0)} ${round3(y0)}`;
```

Comment:
since we're doing the same rounding multiple times, should we store `const rx0 = round3(x0)` -- it's pretty fast though.

Should `A${radius} ${radius}` be `A${round3(radius)} ${round3(radius)}` in the first line also?